Iva-Georgieva/Exercise02-Basic/Basics01/BASICS01-002.js

```diff
@@ -0,0 +1,17 @@
+/* BASICS01-002: Imagine, you are solving a math problem. Declare ten variables, which you might need in your program.
+
+     The math problem to solve:
+         We need to calculate the perimeters of a circle, rectangle and triangle and sum those perimeters.*/
+
+
+const pi; 
```

Comment:
When const variables are being declared, you should assign a value to them. If you run that code you will get
```txt
SyntaxError: Missing initializer in const declaration
```
Change the variable declaration with `let`